src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -621,6 +633,14 @@ describe('non-empty successful affected clusters table', () => {
       // TODO check page is reloaded afterwards
     });
   });
+
+  it('missing impacted date shown as N/A', () => {
+    filterApply({ name: 'e73ee8b2-f60f-4bb0-abc7-be3093debfe7' });
```

Comment:
Would it be possible to get the cluster name from the data? Something like 
```values.filter((v) => !Object.hasOwn(v, 'impacted'))[0].name```?

Same for the cluster with empty impacted

Reply:
good idea, fixed in the last commit

Reply:
awesome, thank you!